model/stacktrace_frame.go

```diff
@@ -80,32 +81,36 @@ func (s *StacktraceFrame) isLibraryFrame(pattern *regexp.Regexp) bool {
 		(s.AbsPath != nil && pattern.MatchString(*s.AbsPath))
 }
 
-func (s *StacktraceFrame) applySourcemap(service Service, mapper sourcemap.Mapper) {
+func (s *StacktraceFrame) applySourcemap(mapper sourcemap.Mapper, service Service, prevFunction string) string {
```

Comment:
@roncohen fyi - this method gets a `prevFunction` string passed in.
If source mapping cannot be applied, the function returns the unchanged `prevFunction` string.
If source mapping is successful, the `prevFunction` is either changed to the new function name or if this is empty to `<unknown>`.